node/tessel-export.js

```diff
@@ -952,6 +957,107 @@ Tessel.LED.prototype.read = function(callback) {
   });
 };
 
+Tessel.Network = {
+  Wifi: function() {
+    var state = {
+      settings: {},
+      connected: false
+    };
+
+    Object.defineProperties(this, {
+      isConnected: {
+        get: function() {
+          return !!state.connected;
+        }
+      },
+      connected: {
+        set: function(value) {
+          state.connected = value;
+        }
+      },
+      settings: {
+        get: function() {
+          return state.settings;
+        },
+        set: function(settings) {
+          util.inherits(state.settings, settings);
```

Comment:
This doesn't work, `util.inherits` expects the second argument to have a `prototype` (which means it needs to be a function). Just use `Object.assign(state.settings, settings)` here